src/classes/HiFiCommunicator.ts

```diff
@@ -187,24 +187,14 @@ export class HiFiCommunicator {
      * isn't supplied as an argument to this function, uses the value of that `token` URL query parameter as the JWT.
      * We should remove that later, because we almost certainly don't want this to stay in the API code, but it's _very_ convenient for sample apps for right now.
      *
-     * @param stackName The WebSocket address to which we make our WebRTC signaling connection is decided based on the following heirarchal logic:
-     * 1. If the code is running in the browser context, and the browser's URL query parameters contains a `?stack=<stackName>` query parameter, 
-     * the WebRTC signaling address will be based off of this `stackName` or `stackURL`. Stack names are used internally by High Fidelity developers when testing new server-side code.
-     *     - If the passed `stackName` is a URL, that URL will be used as the WebRTC Signaling Address directly.
-     * 2. If the code is running in the browser context, and the browser's current location's hostname contains `highfidelity.io` (a hostname for internal use only),
-     * it is very likely that we are running a test in a browser. Tests running in a browser from that hostname should assume that the WebRTC Signaling Address
-     * is at the same host from which the test is served.
-     * 3. If the code is running in the browser context, and our code compilation processes have specified the build mode as "production", we should use the production
-     * WebRTC signaling connection address.
-     * 4. If a developer has passed a `stackName` parameter into this `connectToHiFiAudioAPIServer()` call, use a WebRTC signaling address based on that `stackName`.
-     *     - If the passed `stackName` is a URL, that URL will be used as the WebRTC Signaling Address directly.
-     * 5. If the code is running in the NodeJS context, we will use the "production" `stackName`, and use a WebRTC signaling address based on that `stackName`.
-     * 6. If none of the above logic applies, we will use the default "staging" WebRTC signaling connection address.
+     * @param hostURL An URL that will be used to create a valid WebRTC signaling address. `wws://${hostURL}:8001/?token=`
+     * If the developer does not pass a hostURL parameter, a default URL will be used instead. HiFiConstants.DEFAULT_PROD_HIGH_FIDELITY_ENDPOINT
+     * Reading this parameter from the URL should be implemented by the developer as part of the application code.
```

Comment:
What do you mean by this?

Reply:
This is a note that makes reference to this change. Basically we say that we are not getting information from the URL params anymore. Now we rely on what URL is passed during the connection call. This means that it is the application itself that needs to get this address from the URL params if necessary, so the developer is now in charge of implementing this.

Reply:
oh okay i see!